java/geocoder/pom.xml

```diff
@@ -38,6 +38,26 @@
     </testResources>
     <plugins>
       <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <version>3.2.0</version>
+        <configuration>
+          <instructions>
+            <Export-Package>com.google.i18n.phonenumbers.geocoding</Export-Package>
```

Comment:
I hadn't tested out the geocoder module, I just took the rules that were defined in the previous patch and included them.

Unfortunately, I believe `PhoneNumberToTimeZonesMapper` being in the same package as all of libphonenumber will cause wiring issues within an OSGi environment if it's exported.

Within an OSGi environment each jar/bundle gets it's own ClassLoader. For external imports the environment links these ClassLoaders together based on the Export-Package meta-data. The environment chooses a single ClassLoader per imported package (not class) to link. This would mean that exporting `com.google.i18n.phonenumbers` within the geocoder jar would cause a conflict with the main libphonenumber jar.